tests/director/api-tests/fixtures_request_test.go

```diff
@@ -1,12 +1,12 @@
-package director
```

Comment:
Please consider omitting the `-tests` suffix for `api-tests` and `gateway-integration-tests` directories